.gitignore

```diff
@@ -3,3 +3,5 @@
 coverage/
 Gemfile.lock
 tmp/
+.bundle/
+vendor/
```

Comment:
Did you mean to commit these?

Reply:
I wasn't too sure whether I should git ignore those since they were local dependencies installed and not in original repo, but I think they were already untracked looking at it now, is that right?

Reply:
TBH I am not even sure how you got those folders. Taking a look at mine I don't those folders. I think you can just delete them and remove this change to the gitignore file

Reply:
No worries I'll remove the files locally and update gitignore.

I got them trying to run 'make validate' from the repos [contribution guidelines](https://github.com/TandaHQ/definitions/blob/master/CONTRIBUTING.md).

I got to the bit about running the make validate executable to make test that the updates match the definition format.

```
default: validate

validate:
	bundle exec ruby lib/validation/run.rb

test:
	bundle exec rspec

.PHONY: validate,test
```
When I ran it I had to install dependencies;

```
zak@Zak's-MacBook-Pro definitions % bundle exec ruby lib/validation/run.rb
Could not find gem 'rspec (~> 3)' in any of the gem sources listed in your Gemfile.
Run `bundle install` to install missing gems.
```

After running:

```
zak@Zak's-MacBook-Pro definitions % bundle install
Fetching gem metadata from https://rubygems.org/.......
Resolving dependencies...
Using bundler 1.17.2
Following files may not be writable, so sudo is needed:
  /Library/Ruby/Gems/2.6.0
  /Library/Ruby/Gems/2.6.0/build_info
  /Library/Ruby/Gems/2.6.0/cache
  /Library/Ruby/Gems/2.6.0/doc
  /Library/Ruby/Gems/2.6.0/extensions
  /Library/Ruby/Gems/2.6.0/gems
  /Library/Ruby/Gems/2.6.0/specifications
Fetching coderay 1.1.3

Your user account isn't allowed to install to the system RubyGems.
  You can cancel this installation and run:

      bundle install --path vendor/bundle

  to install the gems into ./vendor/bundle/, or you can enter your password
  and install the bundled gems to RubyGems using sudo.

  Password: 
sudo: a password is required
```

Used the --path vendor/bundle command, but yeah I saw it didn't exist in the repo which did confuse me

Reply:
I would just follow that confluence doc tbh, it's much more up to date and comprehensive